lib/nodes/addon/components/driver-harvester/component.js

```diff
@@ -99,6 +161,7 @@ export default Component.extend(NodeDriver, {
       networks:   get(this, 'globalStore').rawRequest({ url: `${ url }/v1/k8s.cni.cncf.io.networkattachmentdefinition` }),
       namespaces: get(this, 'globalStore').rawRequest({ url: `${ url }/v1/namespace` }),
       configmaps: get(this, 'globalStore').rawRequest({ url: `${ url }/v1/configmap` }),
+      nodes:      get(this, 'globalStore').rawRequest({ url: `${ url }/v1/node` }),
```

Comment:
The topologky key and namespace selects are empty for me: it looks like these requests don't run until I select a different credential from the default (I had two defined before starting this review)

Reply:
topologeyKey field and namespaces field are inputable now, I think it will work expected.
Please take a look @mantis-toboggan-md , thanks!